cedar-policy-core/src/fuzzy.rs

```diff
@@ -0,0 +1,158 @@
+use smol_str::SmolStr;
+
+/// Fuzzy string matching using the Levenshtein distance algorithm
+pub fn fuzzy_search(key: &str, lst: &[impl AsRef<str>]) -> Option<SmolStr> {
```

Comment:
where did this code come from? I feel like we already had Levenshtein algorithm implemented somewhere in Cedar, did you just move it here?  if so, can we delete it from wherever it used to be?

Reply:
Shoot, accidental commit. Was planning to move it to this crate it the process of resolving another issue, but not part of this PR